### PR TITLE
chore(.github/workflows): run all checks even if one fails

### PR DESCRIPTION
**Pull request by thbcmlowk**
* Resolves this [thread](https://github.com/Substra/substra-gha-workflows/pull/5#discussion_r1507307810)
* Proof tested [here](https://github.com/Substra/substra-frontend/actions/runs/8113978023/job/22178565853)

### CODE REVIEW

File: .github/workflows/helm.yml [hunk 1]
```diff
@@ -141,7 +153,7 @@ jobs:
           
             readme-generator-for-helm/bin/index.js -r ${CHARTS_DIR}/README.md -v ${CHARTS_DIR}/values.yaml
             
-            cd ${{inputs.path}}/
+            cd ${{ needs.set-path.outputs.repository_name }}
```
Comment (ThibaultFy) [inline .github/workflows/helm.yml:151]: I don't get this change, did you change the workflow inputs somewhere else ?

Comment (thbcmlowk) [reply .github/workflows/helm.yml:151]: Yes ([here](https://github.com/Substra/substra-gha-workflows/pull/5)) but I forgot this one.

**Comment by guilhem-barthes** [comment]
In your proof, you are publishing even if test is failing, I don't think it is a desired outcome

**Comment by thbcmlowk** [comment]
> In your proof, you are publishing even if test is failing, I don't think it is a desired outcome

Just saw that, indeed it is not. It is fixed in the branch (see: new [proof](https://github.com/Substra/substra-frontend/actions/runs/8113978023/job/22178565853)).

Still need to delete the published charts and republish the 1.0.26...

**Review by guilhem-barthes — APPROVED** [review]

**Pull request merged**